src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
> This mistake is easily spotted in any case (since we include file.h in some other files).

Of course it is easily spotted in a project which compiles everything into a single executable. Nevertheless, consider the following scenario:
1. A mistake of this kind is placed during the implementation of a feature, say it is feature "fA". The implementation is committed in changeset "cA". So far so good.
2. At some time in the future a header from "fA" get included somewhere else and error of this type is easily discovered.
3. In the best case the project history gets unneeded commit which just fixes the header. In reality, most likely, this change will be included in the "cB" changeset which will implement feature "fB". The reader of that change will be puzzled. The readability is screwed.

You see, this is  another example of a technique (`{}` initialisation),  which, if applied systematically, can simplify  life for no cost.